langchain/prompts/prompt.py

```diff
@@ -145,6 +170,22 @@ def from_template(cls, template: str, **kwargs: Any) -> PromptTemplate:
             input_variables=list(sorted(input_variables)), template=template, **kwargs
         )
 
+    @classmethod
+    def from_string(cls, string: str, **partial_variables: Any) -> PromptTemplate:
+        """Load a prompt template from a python string.
+
+        A helper method around from_template that is streamlined specifically
+        for python string templates (e.g., "say {foo}").
+
+        Args:
+            string: a python string to use as the template (e.g., "say {foo}")
+            partial_variables: named arguments to use as partials (e.g., {"foo": "bar"})
+
+        Returns:
+            PromptTemplate instance
+        """
+        return cls.from_template(string, partial_variables=partial_variables)
```

Comment:
Going to remove this -- didn't notice the `**kwargs` in the on ramp above (and needed for output parser). Will keep just the examples